helix-core/src/test/java/org/apache/helix/integration/rebalancer/TestFullAutoNodeTagging.java

```diff
@@ -459,6 +459,9 @@ private boolean withinAverage(Map<String, Integer> countMap, boolean isEmptyAllo
         upperBound = 2;
       }
       int average = computeAverage(countMap);
+      // OR Warning: Handle Error Code
```

Comment:
I don't think the average would be -1 here. Since countMap size has been checked above.
Anyway, in this case, we should return false. Could you please modify it to:
if (average == -1) {
  return false;
}

Reply:
Modification is done.